widgets/tab/tabdemo/tabdemo.go

```diff
@@ -0,0 +1,548 @@
+// Package main provides a system monitor application using tabbed interfaces.
+package main
+
+import (
```

Comment:
A UI nit - I noticed that when a TAB is selected, there is one additional cell highlighted at the beginning of the TAB, i.e. in "| Storage |" the first "|" is also highlighted, while the second isn't. Is that intentional?

Shouls we just highlight the content between the two "| |" ?

Reply:
yes although i fought with this for some time and couldn't get the alignment quite right. I'll need to take this one away for later I think

Reply:
fwiw, we don't have to block the PR on this, it can be fixed later as an improvement either by you or another contributor.